documentation/src/main/asciidoc/topics/ref_cache_store_sfs.adoc

```diff
@@ -19,18 +19,69 @@ defragmented.
 
 .Declarative configuration
 
+* For embedded deployments, {brandname} stores data in the global persistent
+location. Use the `path` attribute to specify a directory that is relative to
+the global persistent location, as in the following example:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<persistence>
+   <file-store path="path/to/myDataStore" max-entries="5000"/>
+</persistence>
+----
+
+{brandname} throws the following exception if you attempt to use a directory
+outside the global persistent location:
+
+----
+ISPN000558: "The store location 'foo' is not a child of the global persistent location 'bar'"
+----
+
+For example, you configure your global persistent location as follows:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<global-state>
+   <persistent-location path="/tmp/example" relative-to="my.data"/>
+</global-state>
+----
+
+To set an absolute path for the Single File store configuration, you must do so as follows:
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<persistence>
+   <file-store path="/tmp/example/path/to/myDataStore" max-entries="5000"/>
+</persistence>
+----
+
+* For server deployments, {brandname} stores data in `{server_home}/data`. You
+should not specify a `path` for Single File cache stores with {brandname}
+server.
+
```

Comment:
This is a bit strong. AFAIK in the server you can still change the location of `globalstate.persistentLocation`, also it would be possible to store in another subdir if desired. For example "path=subDir" will result in the store being located at `{server_home/data/subDir/<cache-name>.dat}.

Also, I'm not sure if the `globalState.persistentLocation` can be configured to exist outside of `{server_home}`. @tristantarrant Can you clarify?

Reply:
@ryanemerson I've done some local testing and changed the global persistent location to be outside the server home dir. it is also possible to set file-store path to a directory relative to the global persistent location. if I set a file-store path to an absolute directory, ISPN080028 gets thrown.